exercises/risk-risiko.cpp

```diff
@@ -27,3 +27,97 @@
   M 3 vs 3 => blue win
   O 2 vs 1 => red win
 */
+
+#include <iostream>
+#include <ctime>
+#include <cstdlib>
+
+using namespace std;
+
+class giocatore {
```

Comment:
consiglio di scrivere sempre il codice in inglese, quindi class Player, inoltre rispetta il PascalCase per i nomi delle classi e camelCase per funzioni o variabili